parquet-column/src/main/java/org/apache/parquet/io/RecordConsumerLoggingWrapper.java

```diff
@@ -112,7 +112,7 @@ public void addBoolean(boolean value) {
      */
     @Override
     public void addBinary(Binary value) {
-      if (DEBUG) log(Arrays.toString(value.getBytes()));
+      if (DEBUG) log(Arrays.toString(value.getBytesUnsafe()));
```

Comment:
I think this is correct. There's no need to copy here unless the backing bytes have an offset and length that doesn't match the byte array's bounds. But in that case, getBytesUnsafe will already make a copy.